R/init.R

```diff
@@ -508,16 +508,57 @@ if (interactive() &&
         )
       }
 
+      print.help_files_with_topic <- function(h, ...) {
+        if (length(h) >= 1 && is.character(h)) {
+          file <- h[1]
+          path <- dirname(file)
+          dirpath <- dirname(path)
+          pkgname <- basename(dirpath)
+          requestPath <- paste0(
+            "/library/",
+            pkgname,
+            "/html/",
+            basename(file),
+            ".html"
+          )
+          request(command = "help", requestPath = requestPath)
+        }
```

Comment:
When the help document is empty (e.g. `?nothing`), nothing happens.

The original behavior is

```
> ?nothing                                    
No documentation for 'nothing' in specified packages and libraries:
you could try '??nothing'
```

Maybe we could do something similar?